pkg/virt-handler/selinux/labels.go

```diff
@@ -114,8 +133,24 @@ func (se *SELinuxImpl) Restore(dir string) (err error) {
 	return nil
 }
 
+func (*SELinuxImpl) InstallPolicy(dir string) (err error) {
+	for _, policyName := range POLICY_FILES {
+		fileDest := dir + "/" + policyName + ".cil"
```

Comment:
Style comment only: In general it's more correct to use `filepath.Join` to combine path elements. It's probably irrelevant. chances of KubeVirt being ported to an OS that uses something other than `/` to separate path elements is virtually nil.